package/gluon-client-bridge/luasrc/lib/gluon/upgrade/310-gluon-client-bridge-local-node

```diff
@@ -19,8 +19,7 @@ uci:section('network', 'device', 'local_node_dev', {
 local ip4, ip6
 
 if site.next_node.ip4 then
-	local plen = site.prefix4:match('/%d+$')
-	ip4 = site.next_node.ip4 .. plen
+	ip4 = site.next_node.ip4 .. '/32'
```

Comment:
The br-client needs a more specific route than the local-node one (/24 vs. /32) as otherwise the kernel would select the wrong interface for outputting de-nat'ed packets coming back.